jupyterhub/handlers/base.py

```diff
@@ -209,6 +223,44 @@ def get_current_user_oauth_token(self):
         self.db.commit()
         return self._user_from_orm(orm_token.user)
 
+    async def refresh_user_auth(self, user, force=False):
+        """Refresh user authentication info
+
+        Calls `authenticator.refresh_user(user)`
+
+        Called at most once per user per request.
+
+        Args:
+            user (User): the user whose auth info is to be refreshed
+            force (bool): force a refresh instead of checking last refresh time
+        Returns:
+            user (User): the user having been refreshed, or None if
```

Comment:
missing end of sentence

Reply:
... the user needs to login again to authenticate.